_topic_map.yml

```diff
@@ -1277,16 +1277,14 @@ Topics:
     File: working-with-pipelines-using-the-developer-perspective
   - Name: OpenShift Pipelines release notes
     File: op-release-notes
-#- Name: GitOps
-#  Dir: gitops
-#  Distros: openshift-enterprise,openshift-webscale,openshift-origin
-#  Topics:
+- Name: GitOps
+  Dir: gitops
+  Distros: openshift-enterprise,openshift-webscale,openshift-origin
+  Topics:
 #  - Name: Understanding OpenShift GitOps
 #    File: understanding-openshift-gitops
-#  - Name: Installing OpenShift GitOps Operator
-#    File: installing-openshift-gitops-operator
-#  - Name: Getting started with OpenShift GitOps
-#    File: getting-satrted-with-openshift-gitops
+   - Name: Getting started with OpenShift GitOps
+     File: getting-satrted-with-openshift-gitops
```

Comment:
typo on my part, hence, the build error. I know I had to look at it with a fresh pair of eyes to find what was wrong :) I'll correct this in my next commit